### PR TITLE
docs(readme): Add additional recommended specs for running a node

### PR DESCRIPTION
**Pull request by wbnns**
Add additional clarity regarding hardware requirements when running a node (specifically with regard to hard drive specifications)

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: README.md [hunk 1]
```diff
@@ -33,7 +33,10 @@ We recommend you have this hardware configuration to run a node:
 
 - a modern multi-core CPU with good single-core performance
 - at least 16 GB RAM (32 GB recommended)
-- a high performance SSD drive (NVME recommended) with at least 750GB (full node) or 4.5TB (archive node) free
+- a locally attached NVMe SSD drive
+- adequate storage capacity to accommodate both the [snapshot restoration process](http://base.org/tutorials/run-a-base-node/#snapshots) (if restoring from snapshot) and chain data, ensuring a minimum of (2 \* current_chain_size) + snapshot_size + 20%\_buffer
```
Comment (0x00101010) [inline README.md:37]: curious why 2 * current_chain_size? We already have 20% buffer specified

**Review by anikaraghu — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (sameersubudhi) [reply README.md:37]: FYI, the specified URL doesn't seem to work. Found: https://docs.base.org/tutorials/run-a-base-node/#snapshots instead.

Comment (wbnns) [reply README.md:37]: @0x00101010 I have seen feedback where people do not have enough space when working with the chain data + snapshot in their local environment so wanted to try to add additional guidance here to hopefully avoid this situation. IMO, I think only the 20% buffer will not provide ample time before encountering disk space issues alongside growth of the chain; PLMK if you think something else might be preferable here 👍 

@sameersubudhi Thanks for catching this; I've dropped this URL -- alongside the typo it looks like NextJS isn't loading the page client side in time to render it correctly with the page anchor

Comment (0x00101010) [reply README.md]: @wbnns Make sense! In this case, can we give 2 recommendations, 1 as minimum (which we call out that it is just barely enough), and 1 recommended with 2 * current_chain_size indicating that it gives people the capability to handle chain growth?

The worry from me is that if user should make their own decision about how many buffer they want to have (cost wise, etc) and we can just give them enough information to make the decision

Comment (Lj2024tx3) [reply README.md]: Ready for review.

**Review by 0x00101010 — APPROVED** [review]

**Pull request merged**